app/src/config.py

```diff
@@ -38,7 +44,9 @@ def get_config_env() -> ConfigType | Literal[False]:
             "ta_token": os.environ["TA_TOKEN"],
             "jf_url": os.environ["JF_URL"],
             "jf_token": os.environ["JF_TOKEN"],
-            "jf_folder": os.environ['JF_FOLDER']
+            "jf_folder": get_variable_or_default(
```

Comment:
That is python built in functionality: https://docs.python.org/3/library/stdtypes.html#dict.get

basically:
```python
os.environ.get("JF_FOLDER", "default_value-here")
```